app/javascript/mastodon/utils/log_out.ts

```diff
@@ -9,8 +7,12 @@ export const logOut = () => {
   methodInput.setAttribute('type', 'hidden');
   form.appendChild(methodInput);
 
-  const csrfToken = Rails.csrfToken();
-  const csrfParam = Rails.csrfParam();
+  const csrfToken = document
+    .querySelector('meta[name=csrf-token]')
```

Comment:
Using `.querySelector<HTMLMetaElement>(…)?.content` should work and make Typescript happier with more correct types

Reply:
Added type hint.